### ISSUE TITLE
In DDPG.V2 the sigmoid saturates, with any scenario

### ISSUE THREAD

**Issue by jandevries123** [MorvanZhou/Reinforcement-learning-with-tensorflow#107]
i have my state all normalized and my rewards as low as ca 0 and 1.
my state looks like:

[ 0.          0.         -1.73       -0.5396034   1.73025715 -0.65105035
 -0.5396034  -0.1368763  -1.15055679  1.28743309]

and my actor critic network (adjusted to discrete) looks like:

    def _build_a(self, s, reuse=None, custom_getter=None):
        trainable = True if reuse is None else False
        with tf.variable_scope('Actor', reuse=reuse, custom_getter=custom_getter):
            net = tf.layers.dense(s, 100, activation=tf.nn.relu, name='l1', trainable=trainable)
            a = tf.layers.dense(net, self.a_dim, activation=tf.nn.tanh, name='a', trainable=trainable)
            return tf.nn.sigmoid(a, name='scaled_a')

i tried some diffrent activation function without luck.

what is the issue?! 

thanks for answering in advance

Jan

**Comment by m5823779** [comment]
Hello, I meet same problem, when my action dimension equal to one, the result is good, but when my action dimension is two (the activation function is tanh and sigmoid), the output of actor will saturate. By the way, I use batch normalization only in my actor network.  And my code reference from: https://github.com/floodsung/DDPG
Have you already solve this problem?

**Comment by jandevries123** [comment]
Please be so kind to delete this issue for me, thank you in advance.
